src/app/server/Mdns.cpp

```diff
@@ -78,6 +78,33 @@ chip::ByteSpan FillMAC(uint8_t (&mac)[8])
 
 } // namespace
 
+uint16_t gSecuredPort   = CHIP_PORT;
+uint16_t gUnsecuredPort = CHIP_UDC_PORT;
+
+/// Sets the secure Matter port
```

Comment:
Doccomments should go in the header. As they stand now, I would consider deleting them. It is obvious that 'SetSecuredPort' is  actually a "sets the secured port" (same for all the other comments below).

It would be even better to document what a secured port is and what an unsecured one is:  when is one used and when is another one used. Should this not be  named something like CommisionableNodePort() and the reverse?

 I am unclear what the reverse is ... the method signature is "bool commisionableNode" so it seems to be a flag. SetCommisionablePort and SetNotCommisionablePort sounds wrong :/

Reply:
addressed via comment updates